ueberdosis/tiptap
# feat: add content to pasted nodes

## Please describe your changes

The `nodePasteRule` exposed by TipTap does not support nested content. This makes it possible to specify the content based on the nodes attributes.

## How did you accomplish your changes

Added an optional parameter similar to `getAttributes` that allows for a type `JSONContent[]` or a function returning `JSONContent[]` to be specified

## How have you tested your changes

Manually. We have reimplemented the `nodePasteRule` in our own codebase for this exact purpose

## How can we verify your changes

Create a node with a paste rule and some content specified. You should see the content as a child when created in the editor

## Remarks

N/A

## Checklist


## Related issues

N/A
